src/server.rs

```diff
@@ -96,10 +96,14 @@ impl Server {
+
         let webui = warp::path::end()
```

Comment:
I think this should be removed, there shouldn't be a need to have `warp::path::end()` two times for a single route.